src/testUtils/mocks/auth.js

```diff
@@ -63,13 +63,13 @@ export const mockUpdateUserFailure = (user) =>
 
 export const mockUpdatePasswordSuccess = (password) =>
   baseMock
-    .patch('/users/password', decamelizeKeys({ password }))
+    .patch('/users', decamelizeKeys({ user: { password } }))
```

Comment:
you can delete these two methods and just use `mockUpdateUserSuccess` by passing the correct parameters